.node-version

```diff
@@ -1 +1 @@
-20.6.0
```

Comment:
We don't support node versions > 20 in actions yet during runtime, should we be bumping this value before that?

Reply:
Good point, I updated this to 20.9.0 since that includes better support for ESM loaders.